homeassistant/components/sensor/strings.json

```diff
@@ -1,26 +1,26 @@
 {
   "device_automation": {
     "condition_type": {
-      "is_battery_level": "{entity_name} battery level",
-      "is_humidity": "{entity_name} humidity",
-      "is_illuminance": "{entity_name} illuminance",
-      "is_power": "{entity_name} power",
-      "is_pressure": "{entity_name} pressure",
-      "is_signal_strength": "{entity_name} signal strength",
-      "is_temperature": "{entity_name} temperature",
-      "is_timestamp": "{entity_name} timestamp",
-      "is_value": "{entity_name} value"
+      "is_battery_level": "{entity_name} battery level is",
```

Comment:
I think that we should make this to be Current entity name battery level. I don't think "is" makes much sense in a drop-down